client/my-sites/earn/ads/wrapper.jsx

```diff
@@ -201,14 +201,14 @@ class AdsWrapper extends Component {
 
 	renderUpsell() {
 		const { siteSlug, translate } = this.props;
-		const bannerURL = `/checkout/${ siteSlug }/premium`;
+		const bannerURL = `/checkout/${ siteSlug }/pro`;
```

Comment:
This has created a path for a legacy plan to transit to the Pro plan, but we will leave it for now: p2-pdgrnI-xR#comment-1052